src/components/AdminDashboard/NotificationsDrawer.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import {
+  Drawer,
+  DrawerBody,
+  DrawerFooter,
+  DrawerHeader,
+  DrawerOverlay,
+  DrawerContent,
+  DrawerCloseButton,
+  useDisclosure,
+  IconButton,
+  Button,
+  Box,
+  Text,
+  VStack,
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+  Badge,
+  useToast,
+} from '@chakra-ui/react';
+import { BellIcon, ChevronDownIcon, ArrowForwardIcon } from '@chakra-ui/icons';
+import { useState } from 'react';
+
+const NotificationsDrawer = ({ notificationsData }) => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const btnRef = React.useRef();
+  const [sortState, setSortState] = useState('');
+  const toast = useToast();
+
+  const notificationAction = {
```

Comment:
Add icons for each notification as shown in the figma, and map them to each notifcation action in the dictionary.
Example: {'New Applicaion': ['View Application', IconObject]...